.github/workflows/pkg-size.yml

```
@@ -0,0 +1,25 @@
+name: Package Size Report
+
+on:
+  - pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions: null
```

actionlint (1.7.4)

"null" is invalid for permission for all the scopes. available values are "read-all" and "write-all"

(permissions)